webpack.config.js

```diff
@@ -99,6 +99,15 @@ module.exports = {
           outputPath: 'media/',
           publicPath: 'bundles/'
         }
+      },
+      // Handles custom fonts. Currently used for icons.
+      {
+        test: /\.woff$/,
+        loader: 'file-loader',
+        options: {
+          name: '[name].[ext]',
+          outputPath: 'app/fonts/',
```

Comment:
This is actually outputting to `bundles/app/fonts` which is a little strange.  I think you want this:
```
outputPath: 'fonts/',
publicPath: 'bundles/'
```
Similar to what we do for media files above.